web/profiles/custom/os2loop/themes/os2loop_theme/templates/content/flag--os2loop-upvote-upvote-button.html.twig

```diff
@@ -19,17 +19,16 @@
 {{ attach_library('flag/flag.link') }}
 
 {# Depending on the flag action, set the appropriate action class. #}
-{# {% if action == 'unflag' %}
-    {% set action_class = 'action-unflag' %}
+{% if action == 'unflag' %}
+    {% set action_class = 'unflag' %}
 {% else %}
-    {% set action_class = 'action-flag' %}
-{% endif %} #}
+    {% set action_class = 'flag' %}
+{% endif %}
```

Comment:
This should be written as `{% set action_class = action == 'unflag' ? 'unflag' : 'flag' %}`